ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -199,6 +200,18 @@ public void setNumLines(ReactEditText view, int numLines) {
     view.setLines(numLines);
   }
 
+  @ReactProp(name = "maxLength")
+  public void setMaxLength(ReactEditText view, @Nullable Integer maxLength) {
+    if (maxLength == null) {
+      InputFilter[] filterArray = new InputFilter[0];
```

Comment:
please extract this as private static constant (e.g. EMPTY_FILTERS) at the top of this class